derp/derp_server.go

```diff
@@ -65,7 +68,7 @@ func init() {
 }
 
 func init() {
-	rand.Seed(time.Now().UnixNano())
+	rand.Seed(clock.Now().UnixNano())
```

Comment:
what's the goal of changing this one? who can change this clock?

Reply:
I think this is unused so maybe I should leave it as is?

Reply:
Yeah, let's keep in as-is. Any test wanting a deterministic seed can call `rand.Seed` again.